src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -24,6 +24,14 @@ struct LogicalImpl;
 /// \ingroup NumericalAlgorithmsGroup
 /// \brief Compute the partial derivatives of each variable with respect to
 /// the logical coordinate.
+///
+/// \requires `DerivativeTags` to be the head of `VariableTags`
+///
+/// \return a `Variables` with a spatial tensor index appended to the front
+/// of each tensor within `u` and each `Tag` wrapped with a `Tags::d`.
```

Comment:
A note: I'm working on changing `Tags::d` to `Tags::deriv`, so maybe it's worth already making that change here so it doesn't get missed